src/components/breadcrumb/Breadcrumb.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <nav :class="breadcrumbClasses">
+        <ul>
+            <template v-for="(item, index) in items">
+                <li
+                    v-if="item"
+                    :key="item"
+                    :class="{ 'is-active': index === items.length - 1 }">
+                    <a
```

Comment:
Please use component as you told you, so the user can use a, router-link or other